SwiftyAttributes/Sources/iOS_tvOS/UIKit+SwiftyAttributes.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  UIKit+SwiftyAttributes.swift
+//  SwiftyAttributes
+//
+//  Created by Roman Podymov on 26/05/19.
+//  Copyright © 2019 Roman Podymov. All rights reserved.
+//
+
+#if os(macOS)
+#elseif os(watchOS)
+#else
+    import UIKit
+    public typealias NavigationBar = UINavigationBar
```

Comment:
I don't think we need the typealias since there's no conflicting navigation bar type